lib/src/components/NxList/__tests__/NxListButtonItem.test.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxListButtonItemProps } from '../types';
+import NxList from '../NxList';
+import { getMountedComponent, getShallowComponent } from '../../../__testutils__/enzymeUtils';
+
+describe('NxListButtonItem', function() {
+
+  const minimalProps: NxListButtonItemProps = {};
+  const getShallow = getShallowComponent(NxList.ButtonItem, minimalProps);
+  const getMounted = getMountedComponent(NxList.ButtonItem, minimalProps);
+
+  it('renders a correctly structured element - a button within an li', function() {
+    const contentEl = getShallow();
+
+    expect(contentEl).toExist();
+    expect(contentEl.find('li')).toMatchSelector('.nx-list__item.nx-list__item--clickable');
```

Comment:
All of these `.find('li')` calls shouldn't be necessary as the `li` should be the top element of the render.  If it fails without them, I would think that indicates that something's wrong

Reply:
> All of these `.find('li')` calls shouldn't be necessary as the `li` should be the top element of the render. If it fails without them, I would think that indicates that something's wrong

`.find('li')` calls have been removed.
96d0bdc629a59a25b03161f65850d7e50e5d8900